bin/yaml/libraries.yaml

```diff
@@ -1489,13 +1489,55 @@ libraries:
         targets:
         - trunk
         type: github
+      ztdcuneicode:
+        build_type: cmake
+        make_targets:
+        - ztd.cuneicode
+        check_file: README.md
+        method: nightlyclone
+        repo: soasis/cuneicode
+        targets:
+        - main
+        type: github
+      ztdencodingtables:
+        build_type: cmake
+        make_targets:
+        - ztd.encoding_tables
+        check_file: README.md
+        method: nightlyclone
+        repo: soasis/encoding_tables
+        targets:
+        - main
+        type: github
+      ztdidk:
+        build_type: cmake
+        make_targets:
+        - ztd.idk
+        check_file: README.md
+        method: nightlyclone
+        repo: soasis/idk
+        targets:
+        - main
+        type: github
+      ztdstaticcontainers:
+        build_type: cmake
+        make_targets:
+        - ztd.static_containers
+        check_file: README.md
+        method: nightlyclone
+        repo: soasis/static_containers
+        targets:
+        - main
+        type: github
       ztdtext:
-        build_type: none
-        check_file: include/ztd/text.hpp
+        build_type: cmake
+        make_targets:
+        - ztd.text
+        check_file: README.md
         method: nightlyclone
         repo: soasis/text
         targets:
-        - trunk
+        - main
```

Comment:
Trunk here is actually the right thing, it doesn't checkout "trunk", it still clones just the default, but the target directory is already "trunk" and its best not to change that. I'm a little hazy if same needs to apply to the new libs.. I don't think so..

Reply:
I think I want "main" directly because I'll be messing with like defaults and `develop` and stuff, so `main` is probably a better idea to be super sure!!